CMakeLists.txt

```diff
@@ -1,24 +1,17 @@
-##############################################################################
-# Convert CMake flags to compiler flags
 ###############################################################################
-if(EVT_CORE_LOG_ENABLE)
-    add_compile_definitions(EVT_CORE_LOG_ENABLE)
-endif()
-
-###############################################################################
-# Top level CMakeList for building the EVT HUDL source code
+# Top level CMakeList for building the EVT BMS source code
 ###############################################################################
 cmake_minimum_required(VERSION 3.15)
 
 set(EVT_CORE_DIR      ${CMAKE_SOURCE_DIR}/libs/EVT-core)
 
-# Link to the EVT-core library
-add_subdirectory(libs/EVT-core/)
-
 include(CMakeDependentOption)
 include(${EVT_CORE_DIR}/cmake/evt-core_compiler.cmake)
 include(${EVT_CORE_DIR}/cmake/evt-core_install.cmake)
 
+# Link to the EVT-core library
+add_subdirectory(libs/EVT-core/)
```

Comment:
Can you explain the problem this change solves? Should the project template be changed to match this?

Reply:
Must've been a mistake. It has been placed back where it was originally.